src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -209,6 +224,11 @@ public Integer getTtl() {
         return cache == null ? null : cache.getTtl();
     }
 
+    @Restricted(NoExternalUse.class)
+    public List<EnvironmentProperty> getExtraEnvVars() {
```

Comment:
EnvVars is something in Jenkins - I would suggest a slightly different name to avoid any confision.

// also a comment on the @restricted // for jelly use only